components/textbox/textbox.tsx

```diff
@@ -297,7 +325,7 @@ export default class Textbox extends React.PureComponent<Props> {
                     onComposition={this.props.onComposition}
                     onBlur={this.handleBlur}
                     onHeightChange={this.handleHeightChange}
-                    style={{visibility: this.props.preview ? 'hidden' : 'visible'}}
+                    style={{visibility: this.props.preview ? 'hidden' : 'visible', display: customEditor ? 'none' : 'block'}}
```

Comment:
This is definitely a hacky way to accomplish. Not sure what the best way to do this would be.

Reply:
Why not swap one component for the other instead of just hiding the invisible ones? I didn't realize that we're still rendering the post textbox when the preview is visible to begin with

Reply:
@hmhealey I was wondering why it was done this way too. I think I ended up doing this as the sizing/placement was already done here and the plugin wouldn't have to be concerned about that. Either way, I've certainly made this uglier, and will check out your suggestion.